content/languages/python/authoring/index.md

```diff
@@ -0,0 +1,252 @@
+---
+kind: tutorial
+languages: [python]
+sidebar: "language:python"
+---
+
+# Python: creating and translating a kata
+
+This article is meant as help for kata authors and translators who would like to create new content in Python. It attempts to explain how to create and organize things in a way conforming to [authoring guidelines](/authoring/guidelines/), the most common pitfalls and how to avoid them.
+
+This article is not a standalone tutorial on creating kata or translations. It's meant to be a complementary, Python-specific part of a more general set of HOWTOs and guidelines related to [content authoring](/authoring/) explaining how to create a Python version of a kata in a way that conforms to authoring guidelines and best practices. If you are going to create a Python translation of a kata, or a new kata in Python from scratch, please make yourself familiar with the aforementioned documents related to authoring in general. 
+
+
+## General info
+
+Any information related to the Python setup on Codewars, the language version, available modules, and setup of the code runner can be found on the [Python reference](/languages/python/) page.
+
+
+## Description
+
+Python code blocks can be inserted with Python-specific part in [sequential code blocks](/references/markdown/extensions/#sequential-code-blocks):
+
+~~~
+```python
+
+...your code here...
+
+```
+~~~
+
+Python-specific paragraphs can be inserted with [language conditional rendering](/references/markdown/extensions/#conditional-rendering):
+
+```
+~~~if:python
+
+...text visible only for Python description...
+
+~~~
+
+~~~if-not:python
+
+...text not visible in Python description...
+
+~~~
+```
+
+## Tasks and Requirements
+
+Some language constructs and features available in Python do not translate well into other languages, and should be avoided if possible:
+- Abuse of dynamic typing, e.g. mixed return types (_"Return the result, or the string 'Error' if no result can be found."_).
+
+Some kata should not be translated into Python, because it can be difficult to keep their initial idea:
+- The Python standard library is very rich and has many utilities available (e.g. `itertools`, combinatorics functions), so some nontrivial requirements in other languages become trivial in Python,
+- Python supports big integers natively, so kata that rely on the implementation of arbitrary precision integer arithmetic become trivial in Python.
+- The Codewars runner provides a set of preinstalled packages which are available for users solving the kata. They can be a real game-changer when it comes to the difficulty of a Python translation. For example, `numpy` can make many matrix manipulation kata much easier.
+
+## Coding
+
+
+### Code style
+
+Python code should stick to generally recognized Python conventions, with [PEP-8](https://www.python.org/dev/peps/pep-0008/) being most widely accepted.
```

Comment:
I'm not sure to understand that sentence properly (I have troubles because of the "with", I believe)

```suggestion
Python code should stick to generally recognized Python conventions, with [PEP-8](https://www.python.org/dev/peps/pep-0008/) being most widely accepted.
```